inst/tests/tests.Rraw

```diff
@@ -13299,8 +13299,7 @@ test(1962.004, duplicated(DT, by = -1L),
      error = 'specify non existing column*.*-1')
 test(1962.005, duplicated(DT, by = 'y'),
      error = 'specify non existing column*.*y')
-test(1962.0061, duplicated(data.table(NULL)), logical(0L))
-test(1962.0062, duplicated(data.table(a = 1L), by = character()), logical())
+test(1962.006, duplicated(data.table(NULL)), logical(0L))
```

Comment:
this `duplicated(., by=character())` was tested against for coverage before, we might count that as a breaking change?

cc @ColeMiller1 

if so, maybe want to sneak it into the upcoming release as "potentially breaking"